src/platform/utilities/feature-toggles/featureFlagNames.js

```diff
@@ -29,4 +29,5 @@ export default Object.freeze({
   vaViewDependentsAccess: 'vaViewDependentsAccess',
   allowOnline1010cgSubmissions: 'allow_online_10_10cg_submissions',
   gibctEybBottomSheet: 'gibctEybBottomSheet',
+  routeStLouisRPOtoBuffaloRPO: 'route_st_louis_rpo_to_buffalo_rpo',
```

Comment:
If my understanding is correct, I think the value is supposed is supposed to be in camelcase, not snakecase. 

>6. Add the feature toggle name (in camel case) to vets-website by updating `featureFlagNames.js`. - https://department-of-veterans-affairs.github.io/veteran-facing-services-tools/platform/tools/feature-toggles/#add-feature

Reply:
Hey @billfienberg,

The documentation does seem to indicate it should be camel case here. I just pushed an update to this branch to change this here. 
This object definition is slightly inconsistent in casing, where `allow_online_10_10cg_submissions` is snake case also for example.

Reply:
I noticed that the keys in this object use both. I don't expect you to change the existing snake case values, but since we do have a documented preference, we should try to adhere to it on new code.

Reply:
I am all for sticking to a documented preference to achieve consistency. Just wanted to provide some insight into my thought process.